packages/data/README.md

```diff
@@ -481,7 +485,7 @@ _Usage_
 ```js
 import { dispatch } from '@wordpress/data';
 
-dispatch( 'my-shop' ).setPrice( 'hammer', 9.75 );
+dispatch( store ).setPrice( 'hammer', 9.75 );
```

Comment:
Since this documentation page starts creating a store with the name `store`, I think it's intuitive that this is the store, right? I didn't want to add imports in these cases because we don't have file names and I think it could make it confusing.

Reply:
it might be clearer if we _do_ have an import, since actual code doing this will have to have the import as well. we can try and make the filename something that's obviously example code:

```ts
import { dispatch } from '@wordpress/data';
import { store as myCustomStore } from 'my-custom-store';

dispatch( myCustomStore ).setPrice( 'hammer', 9.75 );
```

Reply:
By the way the descriptions of these functions are generated from the actual code. Instead of editing it here we need to edit it in the source, which for this one is `data/src/index.js`.

Otherwise this change will be wiped out the next time someone makes a change to the source file and it will be confusing.

Reply:
It would be worth updating the description of the above-mentioned source file (`data/src/index.js`) because it shouldn't say "Given the name of a registered store" but should say something like "Given a store descriptor returns an object of…(also supports legacy calling convention accepting the name of a registered store)."

Reply:
@dmsnell, thank you for the great suggestions! I applied them and I updated the documentation with `npm run docs:build`.

https://github.com/WordPress/gutenberg/pull/41210/commits/2e3fabd83bb5b6fad595fa033b707f83360b5a26

> It would be worth updating the description of the above-mentioned source file (data/src/index.js) because it shouldn't say "Given the name of a registered store" but should say something like "Given a store descriptor returns an object of…(also supports legacy calling convention accepting the name of a registered store)."

https://github.com/WordPress/gutenberg/pull/41210/commits/a79b8a7ea36d9682c030478e6efd48a86dde945a